static-catalog/README.md

```diff
@@ -1,163 +1,215 @@
+
+Catalogs make liberal use of the [rel](https://www.w3schools.com/tags/att_link_rel.asp) attribute on links to describe relationships, so that `Catalog`s and `Item`s always link to where they fit in and what else they are related to.
 
 ### Catalog
 
 A `Catalog` contains links out to `Item`s. Each of the `Catalog`s can link to `Item`s and other downstream `Catalog`s.
 
-
 For example, a catalog of [NAIP imagery](https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/) might look something like this:
 
-_Root catalog_
+_"Root" Catalog_
 ```json
 {
-  "name": "NAIP",
-  "description": "Catalog of NAIP Imagery",
-
-  "license": "PDDL-1.0",
-
-  "links": [
-    { "rel": "self", "href": "naip.json" },
-    { "rel": "child", "href": "naip/30087.json" },
-    ...
-  ],
-
-  "contact": {
-    "name": "Supervisor Customer Services Section",
-    "email": "apfo.sales@slc.usda.gov",
-    "phone": "801-844-2922",
-    "url": "https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/"
-  },
-
-  "formats": ["geotiff", "cog"],
-
-  "keywords": ["aerial"],
-  "homepage": "http://wherever",
-
-  "provider": {
-      "scheme": "s3",
-      "region": "us-east-1",
-      "requesterPays": "true"
-  }
+    "name": "NAIP",
+    "description": "Catalog of NAIP Imagery",
+
+    "license": "PDDL-1.0",
+
+    "links": [
+        { "rel": "self", "href": "naip.json" },
+        { "rel": "child", "href": "naip/30087.json" },
+        ...
+    ],
+
+    "contact": {
+        "name": "Supervisor Customer Services Section",
+        "email": "apfo.sales@slc.usda.gov",
+        "phone": "801-844-2922",
+        "url": "https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/"
+    },
+
+    "formats": ["geotiff", "cog"],
+
+    "keywords": ["aerial"],
+    "homepage": "http://wherever",
+
+    "provider": {
+        "scheme": "s3",
+        "region": "us-east-1",
+        "requesterPays": "true"
+    }
 }
 ```
 
-A 'Root Catalog' should contains all the metadata that applies to all items in its catalog. It is typically where
-the contact information goes, as well as links to the products contained. It usually links to one or more 'Linked Catalogs',
-who inherit its catalog level metadata and in turn link to other catalogs or items. A relatively small Root Catalog could
-also choose to only link to items, with no Linked Catalogs.
+A root `Catalog` should contain all the metadata that applies to all items in its catalog. It is typically where the contact information goes, as well as links to the items it contains. It usually links to one or more `catalogs`, who inherit its catalog-level metadata and in turn link to other `catalog`s or `item`s.
+A root `catalog` could also choose to include `item`s, with `catalog` links.
+Below is an example of a sub-`catalog` (that would inherit metadata from It's parent)
 
-_Linked Catalog_
+_"Child" Catalog_
```

Comment:
The example child catalogs should probably have the `root` catalog link defined.